docs/features.md

```diff
@@ -0,0 +1,315 @@
+     }
+   ]
+ }
+}
+```
+
+- **PROTECTED_RESOURCES** – is used to block the edit and delete functions based on the determined rules. If the resource meets the rule requirements, the resource becomes protected and cannot be edited/deleted.  
+  Each resource must have a match field, which is the list of key-value pairs. The proper rule description is when the definition given in the key matches the value.
+
+  Optionally, you can provide a `message` parameter, which displays a simple message, or `messageSrc`, which is a yaml path where the message to be displayed is included. If neither message nor messageSrc is provided, a generic message will be used.
+
+Example:
+
+```bash
+"PROTECTED_RESOURCES": {
+  "isEnabled":true,
+  "config": {
+    "resources": [
+      {
+        # matches a resource with label "foo" equal to "bar"
+        "match": {
+          "$.metadata.labels.foo": "bar"
+        },
+        # message to display in the tooltip
+        "message": "This resource is protected"
+      },
+      {
+        # matches a resource with both label "foo" equal to "bar" and label "baz" equal to "qux"
+        "match": {
+          "$.metadata.labels.foo": "bar",
+          "$.metadata.labels.baz": "qux"
+        },
+        # source of the message to display in the tooltip
+        "messageSrc": "$.metadata.annotations.protected-message"
+      },
+      {
+        "match": {
+          "$.metadata.labels.foo": "bar"
+        }
+        # if neither "message" or "messageSrc" is given, a generic message will be used (from i18n files - key: common.protected-resource)
+      }
+    ]
+  }
+}
+```
+
+The match keys and messageSrc must use format as described at
+<https://github.com/dchester/jsonpath>.
+
+- **SENTRY** – is used to enable monitoring of uncaught exceptions, which then are analyzed and repaired. The address to which you send the information is located under the dsn key.
+
+Default settings:
+
+```bash
+"SENTRY ":{
+  "isEnabled": false,
+  "selectors":[ ],
+  "config": {
+    "dsn":""
+  }
+},
+```
+
+- **SERVERLESS** – is used to show or hide the Serverless view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.  
+  For the view to be shown, you must enable the feature and all the CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"SERVERLESS ":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"serverless.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **SERVICE_CATALOG** – is used to show or hide the Service Catalog views (Catalog, Instances, and Brokers) as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.  
+  SERVICE_CATALOG and SERVICE_CATALOG_ADDONS are also used to determine if Service Bindings (in Configuration tab) and environment variables injected by Service Bindings (in Code Tab) are displayed in the Functions view.  
```

Comment:
I would use bold for **SERVICE_CATALOG** and  **SERVICE_CATALOG_ADDONS** or wouldn't use bold for any of the listed features.
```suggestion
  SERVICE_CATALOG and SERVICE_CATALOG_ADDONS are also used to determine if Service Bindings (in the **Configuration** tab) and environment variables injected by Service Bindings (in the **Code** tab) are displayed in the Functions view.  
```